Lib/unittest/test/testmock/testpatch.py

```diff
@@ -1854,5 +1855,25 @@ def foo(*a, x=0):
             self.assertEqual(foo(), 1)
         self.assertEqual(foo(), 0)
 
+    def test_patch_dict_with_orderdict(self):
```

Comment:
Note this is also testing patch of ordered dicts. You might be able to find a better name.

Reply:
Hi @mariocj89 I think this was resolved